tests/integration-service/integration.go

```diff
@@ -455,7 +466,7 @@ func createComponent(f framework.Framework, testNamespace, applicationName, comp
 		},
 	}
 
-	originalComponent, err := f.AsKubeAdmin.HasController.CreateComponent(componentObj, testNamespace, "", "", applicationName, false, utils.MergeMaps(utils.MergeMaps(constants.ComponentPaCRequestAnnotation, constants.ImageControllerAnnotationRequestPublicRepo), constants.DefaultDockerBuildPipelineBundle))
+	originalComponent, err := f.AsKubeAdmin.HasController.CreateComponent(componentObj, testNamespace, "", "", applicationName, false, utils.MergeMaps(utils.MergeMaps(constants.ComponentPaCRequestAnnotation, constants.ImageControllerAnnotationRequestPublicRepo), buildPipelineAnnotation))
```

Comment:
Can you please also add this change within the GitLab test suite of integration service [here](https://github.com/konflux-ci/e2e-tests/blob/cfef6ad9b2419f76a8fe2f42a8927a14508897ef/tests/integration-service/gitlab-integration-reporting.go#L126)?

Reply:
Good catch. Added in the latest commit.